docs: using Dyno (#1056)
elastic/apm-integration-testing: README.md

|Node|:black_square_button:|
|Ruby|:black_square_button:|

### Using Dyno with a remote APM Server

It is possible to connect the infrastructure generated by compose.py to a remote APM Server.

> This makes possible to send APM date from the scenarios modelled with Dyno.

Reviewer: ```suggestion
This makes possible to send APM data from the scenarios modeled with Dyno.
```

The following command will launch Opbeans-python and Dyno locally,
the Opbeans-python is configured as the environment `local` of the service `dyno-service`,
and using the version name `test-demo`

```
APM_SERVER_URL=https://apm.example.com \
APM_TOKEN=MySuPerApMToKen \
python3 ./scripts/compose.py start 8.0.0 \
  --no-kibana
  --no-elasticsearch \
  --dyno  \
  --apm-server-url "${APM_SERVER_URL}" \
  --apm-server-secret-token="${APM_TOKEN}" \
  --with-opbeans-python \
  --opbeans-python-service-environment local \
  --opbeans-node-service-version test-demo \
  --opbeans-python-service-name dyno-service
```

when the Docker container started you can connect to the Dyno UI at http://localhost:9000 and modify the scenario to cause errors,
in this case we have disabled PostgreSQL to cause a database service error.

> ![](images/dyno.png)

Author: ```suggestion
![](images/dyno.png|width=400)
```

> ![](images/dyno.png|width=400)

Author: ```suggestion
![](images/dyno.png|width=400px)
```

> ![](images/dyno.png|width=400)

Author: ```suggestion
![](images/dyno.png | width=400)
```

> ![](images/dyno.png | width=400)

Author: ```suggestion
![](images/dyno.png)
```

> ![](images/dyno.png|width=400px)

Author: ```suggestion
![](images/dyno.png | width=400)
```

Then we can check the result of our changes in Dyno in the APM UI, in this case we can see that
the error rate for postgreSQL is increasing.

> ![](images/apm-dyno.png)

Author: ```suggestion
![](images/apm-dyno.png|width=400)
```

> ![](images/apm-dyno.png|width=400)

Author: ```suggestion
![](images/apm-dyno.png | width=400)
```

> ![](images/apm-dyno.png | width=400)

Author: ```suggestion
![](images/apm-dyno.png)
```
